Tests/HTTPKitTests/HTTPTests.swift

```diff
@@ -33,6 +33,19 @@ class HTTPTests: HTTPKitTestCase {
         
         XCTAssertEqual(httpReq.headers.firstValue(name: .cookie), "id=value")
     }
+    
+    func testMultipleCookiesAreSerializedCorrectly() throws {
```

Comment:
This test needs to be added to the `allTests` array here: https://github.com/vapor/http/blob/master/Tests/LinuxMain.swift

Reply:
Fixed